lib/mysql2/client.rb

```diff
@@ -18,6 +18,9 @@ def self.default_query_options
       }
     end
 
+    # rubocop:disable Metrics/AbcSize
+    # rubocop:disable Metrics/CyclomaticComplexity
+    # rubocop:disable Metrics/MethodLength
```

Comment:
Oh, no, please increase the limits in .rubocop_todo.yml - or just remove this, I need to update the limits for another pending PR anyways.